scripts/build.sh

```diff
@@ -0,0 +1,47 @@
+# Create a browser extension release package
+
+# 1. Capture browser type from argument
+browserType=$1
+
+# 2. Construct release zip filename
+zipName=$browserType-release.zip
+
+# 3. Ensure release directory exists (create if needed)
+mkdir -p build/release/
+
+# 4. Clean up any existing release zip
+rm -f build/release/$browserType-release.zip
```

Comment:
I think you can move steps 3 and 4 down below where you need them, right? No need to do them here?

Reply:
> I think you can move steps 3 and 4 down below where you need them, right?

You're totally right!